Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -140,15 +140,15 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
 ```bash
 exec "$SHELL"
 ```
-4. Now you can install Python 3.8.15 and the associated pip like this:
+4. Now you can install Python 3.9 and the associated pip like this:
 
    ```console
-   $ pyenv install 3.8.15
-   installing python-3.8.15...
+   $ pyenv install 3.9
+   installing python-3.9...
    patching file misc/news.d/next/build/2021-10-11-16-27-38.bpo-45405.isfdw5.rst
```

Comment:
I think you can drop this line and the next two as well.

Reply:
I took your comment literally. I erased lines 148-150. If this is not what you meant please let me know.

Reply:
Yes that is correct. When I followed these steps I didn't get the output on lines 148-150.